### ISSUE TITLE
Possible broken links- Amazon searching for Dungeons and Dragons???

### ISSUE THREAD

**Issue by lokixx** [jef/streetmerchant#1748]
## Expected Behavior

Amazon should be looking for 3080s

## Current Behavior

Amazon looks for 3080s but also noticed an error log to a Dungeons and Dragons item (See error code below)

## Environment

- OS:Windows 10

```# ** All configuration variables are optional **
# Copy this file and name it as `dotenv` (without backticks)
# Read https://jef.codes/streetmerchant/getting-started/#customization for help$
###############################################################################$
INCOGNITO=false
BROWSER_TRUSTED=false
HEADLESS=false
AUTO_ADD_TO_CART=true
COUNTRY=usa
DISCORD_NOTIFY_GROUP=
DISCORD_WEB_HOOK=
EMAIL_PASSWORD=
EMAIL_TO=
EMAIL_USERNAME=a@b.com
LOG_LEVEL=info
MAX_PRICE_SERIES_3080=1300
OPEN_BROWSER=true
PHONE_CARRIER=
PHONE_NUMBER=
PROXY_ADDRESS=
PROXY_PROTOCOL=
PROXY_PORT=
SHOW_ONLY_BRANDS=evga,asus,msi
SHOW_ONLY_SERIES=3080,3060ti
STORES="amazon,bestbuy,newegg,adorama,bandh,asus"
WEB_PORT=6601
IN_STOCK_WAIT_TIME=30
LOW_BANDWIDTH=false
PAGE_BACKOFF_MIN=10000
PAGE_BACKOFF_MAX=3600000
PAGE_SLEEP_MIN=10000
PAGE_SLEEP_MAX=15000
PAGE_TIMEOUT=0
NVIDIA_ADD_TO_CART_ATTEMPTS=10
NVIDIA_SESSION_TTL=60000
SCREENSHOT=false

```

## Logs

```[3:02:02 PM] error :: net::ERR_FAILED at https://www.amazon.com/s?k=dungeons+and+dragons&i=todays-deals&ref=nb_sb_noss_2[3:02:02 PM] error :: ✖ [amazon] msi 3080 gaming x trio - net::ERR_FAILED at https://www.amazon.com/dp/B08HR7SV3M```

**Comment by wofnull** [comment]
Included in Update 09827741aec2158e88c19fd6b0156c32736671c8 ( 3days ago ) from @neatchee was an fix for amazon to prevent the bot protection / captcha protection from amazon using a random search for dungeons and dragons and reverting back to normal site parsing directly after.

so i think the log entry is fine. Howefer net::ERR_FAILED for the 3080 and the search looks more like a general connection issue to amazon. is there a proxy in use?

**Comment by neatchee** [comment]
@wofnull is correct; that error is indicating that the captcha-deterrent - which loads another page on the site between each product check - has failed; I'd definitely be interested to get some more information on your setup to see if we can figure out why that page failed to load (and if it killed the rest of the product check)
